src/test/java/org/broadinstitute/hellbender/tools/copynumber/CreateBinningIntervalsIntegrationTest.java

```diff
@@ -0,0 +1,221 @@
+package org.broadinstitute.hellbender.tools.copynumber;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.engine.ReferenceDataSource;
+import org.broadinstitute.hellbender.utils.GenomeLocParser;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import htsjdk.samtools.util.IntervalList;
+import htsjdk.samtools.util.Interval;
+
+import java.io.File;
+import java.io.*;
+import java.io.IOException;
+
+
+public class CreateBinningIntervalsIntegrationTest extends CommandLineProgramTest {
+
+    private static final File REFERENCE = new File(b37_reference_20_21);
+    private static final GenomeLocParser GLP = new GenomeLocParser(ReferenceDataSource.of(REFERENCE).getSequenceDictionary());
+
+    // Test for separate intervals
+    @Test
```

Comment:
You have some code duplication in all the test cases. You can move all the arguments for test cases into a `DataProvider` and have a single test method that will run on all test cases in the `DataProvider`

Reply:
Thanks for the comment. I created a DataProvider for most of my tests.